### ISSUE TITLE
Reader not loading pages correctly - sorting some pages incorrectly

### ISSUE THREAD

**Issue by lalalasupa0** [null2264/yokai#78]
### Steps to reproduce

Open chapter with many pages (idk about the ones with regular amount of pages)
Scroll down (neither slowly nor too fast)

### Expected behavior

Pages should be in web order

### Actual behavior

Pages get messed up, sometimes page number 27 appearing where number 41 should be

### Yōkai version

1.8.2-b5148

### Android version

Android 12

### Device

Samsung S10

### Other details

Happens to me in chapters with many pages

Exiting the chp. or swiping back is supposed to fix it, but it keeps happening later with other pages

Also happens with downloades chapters (online too, of course)

Using Nightly Build

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, or a request for an extension, I should be contacting the extensions repository's maintainer/support for help.
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Mihon](https://github.com/mihonapp/mihon)
- [X] I have tried the [troubleshooting guide](https://mihon.app/help/).
- [X] I have updated the app to version **[1.8.2](https://github.com/null2264/yokai/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by NoChain69** [comment]
Yeah, suffering from this problem too. Thought it was the sources, but when looking in Mihon, it loads up perfectly without problem. I think the new updates messed it up.

**Comment by null2264** [comment]
Can any of you send me name of affected title and source? I couldn't replicate this issue with the sources I usually used.

**Comment by NoChain69** [comment]
> Can any of you send me name of affected title and source? I couldn't replicate this issue with the sources I usually used.

As the most recent example, I used MangaFire source and read "The Emperor's Sword" Chapter 55.

**Comment by lalalasupa0** [comment]
> Can any of you send me name of affected title and source? I couldn't replicate this issue with the sources I usually used.

It happens a lot more in "chapters" with many vertically thin pages, try in the e-hentai source with some games cg gslleries or something like that, I haven't tried it but you should get the issue there
The important thing is that it is more noticeable the more pages there are and the thinner they are

**Comment by null2264** [comment]
Interesting...

**Comment by NoChain69** [comment]
The new update doesn't seem to fix it, I found out that if you enable the zoom out option and zoom out enough you can see the panel but it becomes very glitch-y.
